src/phpDocumentor/Console/Command/Project/RunCommand.php

```diff
@@ -269,6 +277,13 @@ protected function execute(InputInterface $input, OutputInterface $output) : int
         $output->writeln('phpDocumentor ' . $this->getApplication()->getVersion());
         $output->writeln('');
 
+        try {
+            $this->releaseChecker->check(Application::VERSION()->getShortVersion());
```

Comment:
maybe we should provide an option to disable this feature on the commandline. For ci environments without an internet connection.